### DISCUSSION TITLE
Open Source the code?

### DISCUSSION THREAD

**Discussion by ccthiel** [fmasa/wfrp-master#37] [General]
Noticed you're not charging anything nor are you placing ads (super awesome of you! ). Ever considered just opening up your repo for other people to submit PRs against?

**Comment by ccthiel** [comment]
... I'm a dumbass. I thought this was just a wiki and totally missed the code tab.

**Comment by fmasa** [comment]
Previously all the code was proprietary with ads and paid ad-removal.

But from 3.2.0 onwards everything is free and OSS and I plan to keep it that way 🙂

I plan to prepare Wiki for potential contributors as well.

**Comment by ccthiel** [comment]
This is awesome!  Thanks so much for all the work!
